addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -2850,692 +2850,6 @@ return Timers;
-     */
-    _renderMessageNotificationPopover(messages) {
-        if (this._messageMailPopover) {
-            this._messageMailPopover.popover('hide');
-        }
-        if (!this.$('.o_thread_tooltip').length) {
-            return;
-        }
-        this._messageMailPopover = this.$('.o_thread_tooltip').popover({
-            html: true,
-            boundary: 'viewport',
-            placement: 'auto',
-            trigger: 'hover',
-            offset: '0, 1',
-            content: function () {
-                var messageID = $(this).data('message-id');
-                var message = _.find(messages, function (message) {
-                    return message.getID() === messageID;
-                });
-                return QWeb.render('im_livechat.legacy.mail.widget.Thread.Message.MailTooltip', {
-                    notifications: message.getNotifications(),
-                });
-            },
-        });
-    },
-    /**
-     * Render the popover when mouse hovering on the seen icon of a message
-     * in the thread. Only seen icons in non-squashed message have popover,
-     * because squashed messages hides this icon on message mouseover.
-     *
-     * @private
-     * @param {im_livechat.legacy.mail.model.AbstractThread} thread with thread seen mixin,
-     *   @see {im_livechat.legacy.mail.model.ThreadSeenMixin}
-     * @param {im_livechat.legacy.mail.model.Message[]} messages list of messages in the
-     *   rendered thread.
-     */
-    _renderMessageSeenPopover: function (thread, messages) {
-        var self = this;
-        this._destroyOpenSeenPopoverIDs();
-        if (this._messageSeenPopover) {
-            this._messageSeenPopover.popover('hide');
-        }
-        if (!this.$('.o_thread_message_core .o_mail_thread_message_seen_icon').length) {
-            return;
-        }
-        this._messageSeenPopover = this.$('.o_thread_message_core .o_mail_thread_message_seen_icon').popover({
-            html: true,
-            boundary: 'viewport',
-            placement: 'auto',
-            trigger: 'hover',
-            offset: '0, 1',
-            content: function () {
-                var $this = $(this);
-                self._openedSeenPopoverIDs.push($this.attr('aria-describedby'));
-                var messageID = $this.data('message-id');
-                var message = _.find(messages, function (message) {
-                    return message.getID() === messageID;
-                });
-                return QWeb.render('im_livechat.legacy.mail.widget.Thread.Message.SeenIconPopoverContent', {
-                    thread: thread,
-                    message: message,
-                });
-            },
-        });
-    },
-    /**
-     * @private
-     */
-    _updateTimestamps: function () {
-        var isAtBottom = this.isAtBottom();
-        this.$('.o_mail_timestamp').each(function () {
-            var date = $(this).data('date');
-            $(this).html(mailUtils.timeFromNow(date));
```

Comment:
ci/security due to this move

Reply:
Looks wrong to use html in this case, timeFromNow is just a string, should set element text instead?

Reply:
Indeed it's strange to use `html`, but keep in mind PR is just a move.

Reply:
Fair enough, but should make a prep PR to keep minimal diff here then, not the other way around.